README.rst

```diff
@@ -37,35 +37,45 @@ Some noteworthy flags::
 
     codespell -I FILE, --ignore-words=FILE
 
-The -I flag can be used to whitelist certain words that are in the ``codespell_lib/data/dictionary.txt``. The format of the whitelist file is one word per line. Invoke using: ``codespell -I path/to/file.txt`` to execute codespell referencing said whitelist. **Important note:** The whitelist passed to ``-I`` is case-sensitive based on how it is listed in ``dictionary.txt``. ::
+The ``-I`` flag can be used to whitelist certain words that are in the ``codespell_lib/data/dictionary.txt``. The format of the whitelist file is one word per line. Invoke using: ``codespell -I path/to/file.txt`` to execute codespell referencing said whitelist. **Important note:** The whitelist passed to ``-I`` is case-sensitive based on how it is listed in ``dictionary.txt``. ::
 
     codespell -L word1,word2,word3,word4
 
-The -L flag can be used to whitelist certain words that are comma-separated placed immediately after it. ::
+The ``-L`` flag can be used to whitelist certain words that are comma-separated placed immediately after it. ::
 
     codespell -S, --skip=
 
-Comma-separated list of files to skip. It accepts globs as well.  Examples:
+The ``-S`` (``--skip``) flag is used to skip over a comma-separated list of files and/or directories. It accepts globs as well. **Important note:** Do not append a ``/`` (forward-slash) to a directory you want to skip, this renders the entry for that particular directory ineffective. ::
 
-* to skip .eps & .txt files, invoke ``codespell --skip="*.eps,*.txt"``
+The ``-i <number>`` flag is used to invoke 'interactive mode' when writing changes. 0: no interactivity. 1: ask for confirmation. 2: ask user to choose one fix when more than one is available. 3: will combine both options 1 and 2 together.
 
-* to skip directories, invoke ``codespell --skip="./src/3rd-Party,./src/Test"``
+Examples:
 
+* Skip .eps & .txt files, can be invoked either way::
+
+    codespell --skip="*.eps,*.txt"
+    codespell -S *.eps,*.txt
+
+* Skip directories, can be invoked either way::
+
+      codespell --skip="src/3rd-Party,src/Test"
+      codespell -S src/3rd-Party,src/Test
 
 Useful commands::
 
-    codespell -d -q 3 --skip="*.po,*.ts,./src/3rdParty,./src/Test"
+    codespell -d -q 3 --skip="*.po,*.ts,src/3rdParty,src/Test"
 
 List all typos found except translation files and some directories.
-Display them without terminal colors and with a quiet level of 3. ::
+Display the results without terminal colors (``-d``). 
+The ``-q 3`` means we're running codespell with a quiet level of 3, which represents not to notify us that it's skipping binary files. ::
 
     codespell -i 3 -w
 
-Run interactive mode level 3 and write changes to file.
+Run ``-i 3`` is interactive mode level 3 (see explanation above) and ``-w`` will write changes to file. **Note**: Omitting the ``-w`` flag will mean that your changes will not be saved!  
```

Comment:
Again this feels like a bug, this system should at least log a message and perhaps just exit if you select interactive without write (I guess the only valid exception would be --dry-run).